src/fluree/db/query/exec/eval.cljc

```diff
@@ -115,12 +116,16 @@
   [s substr]
   (str/ends-with? s substr))
 
-(def allowed-filters
-  '#{&& || ! > < >= <= = + - * / and bound coalesce if nil?
-     not not= now or re-find re-pattern strStarts strEnds})
+(defn subStr
+  [s start end]
+  (subs s start end))
+
+(def allowed-fns
+  '#{&& || ! > < >= <= = + - * / quot and bound coalesce if nil?
+     not not= now or re-find re-pattern strStarts strEnds subStr})
 
 (def allowed-symbols
-  (set/union allowed-aggregates allowed-filters))
+  (set/union allowed-aggregates allowed-fns))
```

Comment:
This is pretty minor, but we might need a better name than `allowed-fns`, because all of the aggregates are also functions; they just act on different kinds of inputs (grouped values vs individual values). It doesn't matter in practice, but it might make the code a little more confusing for the uninitiated. I get why "filters" is no longer a good name because it's to specific. Those functions now are used in more than just filters, but I think "fns" is too general. I can't think of a better name right now, but I'll noodle over it.

Reply:
What do you think of my proposal in [e8e2945](https://github.com/fluree/db/pull/397/commits/e8e294559c0aee3b5a1d87bbf24853f988b23877)?

Reply:
perfect!